### PR TITLE
[FLOC-2479] Add UserVoice to the README.

### PR DESCRIPTION
**Pull request by robhaswell**
https://clusterhq.atlassian.net/browse/FLOC-2479

### CODE REVIEW

**Comment by davidoliverSP2** [comment]
looks good. Thanks @robhaswell 

I'll get one of the Engineering team to check it 1st thing before merging

**Comment by jongiddy** [comment]
Feature Requests should appear after Documentation. We would like users to at least read the docs before asking for additional things.

**Comment by jongiddy** [comment]
This should be a general Support section, with a link to the support web page at https://clusterhq.com/flocker/support/ and then perhaps the feature requests line.  Getting help with using the existing software is more important to users than having the chance to ask for something different.

**Comment by adamtheturtle** [comment]
This should go in the http://doc-dev.clusterhq.com/gettinginvolved/contributing.html#talk-to-us and maybe  http://doc-dev.clusterhq.com/labs/index.html#getting-in-touch-with-clusterhq-labs.

File: README.rst [hunk 1]
```diff
@@ -35,6 +36,15 @@ Documentation
 You can read more about `installing Flocker`_, follow a `tutorial`_ and learn about the `features of Flocker and its architecture`_ in the docs.
 
 
+Support
+-------
+
+If you need support to use Flocker, please check the `Flocker Support page`_.
+
+If you have any feature requests or suggestions, we would love to hear about them.
+Please add your ideas to our `UserVoice`_ forum, or file a `GitHub issue`_.
```
Comment (adamtheturtle) [inline README.rst]: I don't think that it is useful to call this "UserVoice" - that is an implementation detail and not mentioned in the linked-to page.

File: README.rst [hunk 2]
```diff
@@ -35,6 +36,15 @@ Documentation
 You can read more about `installing Flocker`_, follow a `tutorial`_ and learn about the `features of Flocker and its architecture`_ in the docs.
 
 
+Support
```
Comment (adamtheturtle) [inline README.rst]: This section overlaps / contradicts the line a few lines above:

>  If you have any issues or feedback, you can talk to us

That links to the talk to us page.

**Comment by adamtheturtle** [comment]
This is not mentioned on the New Starters Checklist, and should be if we are telling users to get in touch via this system. Also useful would be to know who is responsible for managing this forum.

File: README.rst (hunk 1 above)
Comment (adamtheturtle) [inline README.rst]: Also we have an external contributor workflow in JIRA.

**Comment by adamtheturtle** [comment]
"If you have any feature requests or suggestions" surely this is covered by "we're also happy to accept `GitHub issues`_ with feature requests"?

**Comment by adamtheturtle** [comment]
Imagining myself as an external reader of this README who wants to suggest a feature - this just provides confusion as to whether I use GitHub or UserVoice.

**Comment by adamtheturtle** [comment]
I have updated this PR (feel free to revert if the changes are not suitable).

**Comment by adamtheturtle** [comment]
Even if this is merged this needs the changes suggested above (internal docs of who needs to manage this site and updates to the new starters guide) so please do not mark the issue as Done.

**Comment by jongiddy** [comment]
I like the simplicity of your changes. However, it will require a docs release to make this work.  I suggest that for DockerCon, we add a note to `README` and add a follow-up issue to simplify.

**Comment by robhaswell** [comment]
> This is not mentioned on the New Starters Checklist, and should be if we are telling users to get in touch via this system. Also useful would be to know who is responsible for managing this forum.

This does not need mentioning in the New Starters' Checklist, in fact we have reached the limit of the number of admins there can be.

Thanks everyone.

**Pull request merged**